Updates the New visualization info (#3603)
SumoLogic/sumologic-documentation: docs/search/get-started-with-search/search-basics/chart-search-results.md

For this example, you can see a bar chart, but you can pick from any of the available charting options, see [Chart Panel Types](/docs/dashboards/panels) for details.

### New Visualization

The **New Visualization** feature enables the new chart visualizations for your log search, allowing you customize each chart by modifying the available settings and providing a unified experience across the Sumo Logic platforms.

* **Data**. This section represents the data that has been used to plot the chart. Toggle the **Round data point** ON/OFF the round off to the value in tooltip,.

> * **Visual Settings**. These options allow you to further customize your chart: [**Color Palette**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), [**Fill Opacity**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), [**Marker Type**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), and [**Dimensions**](/docs/dashboards/panels/modify-chart/#modifychart-axes) (x, y, and z).

Author: ```suggestion
* **Visual Settings**. Allows you to further customize your chart. The other customizing properties include: [**Color Palette**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), [**Fill Opacity**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), [**Marker Type**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), and [**Dimensions**](/docs/dashboards/panels/modify-chart/#modifychart-axes) (x, y, and z).
```

### New Visualizations

> New Visualizations will enable the new chart visualizations for your log search, also helping you to customize each chart by modifying the available settings and providing a unified experience across the Sumo Logic platforms.
> 
> **Data** represents the data that has been used to plot chart, **Round data point** toggle button is used to turn ON/OFF the round off to the value in tooltip, and the **Visual Settings** options allow you to customize your chart. The other customizing properties include **[Display Overrides](/docs/dashboards/panels/modify-chart/#overridedashboard-displays)**, **[Axes](/docs/dashboards/panels/modify-chart/#modifychart-axes)**, **[Legend](/docs/dashboards/panels/modify-chart/#modifychart-legend)**, **Thresholds**, and **[JSON](/docs/dashboards/panels/modify-chart/#modifychart-legend)**.
> 
> <img src={useBaseUrl('img/search/get-started-search/search-basics/log-query-charts-properties.png')} alt="log-query-charts-properties" style={{border:'1px solid black'}} width="800" />

Reviewer: ```suggestion
The **New Visualization** feature enables the new chart visualizations for your log search, allowing you customize each chart by modifying the available settings and providing a unified experience across the Sumo Logic platforms.

* **Data**. This section represents the data that has been used to plot the chart. Toggle the **Round data point** ON/OFF the round off to the value in tooltip,.
* **Visual Settings**. These options allow you to further customize your chart: **Color Palette**, **Fill Opacity**, **Marker Type**, and **Dimensions** (x, y, and z).

<! -- insert New Visualizations screenshot -->
```

### New Visualizations

The **New Visualization** feature enables the new chart visualizations for your log search, allowing you customize each chart by modifying the available settings and providing a unified experience across the Sumo Logic platforms.

* **Data**. This section represents the data that has been used to plot the chart. Toggle the **Round data point** ON/OFF the round off to the value in tooltip,.
* **Visual Settings**. These options allow you to further customize your chart: **Color Palette**, **Fill Opacity**, **Marker Type**, and **Dimensions** (x, y, and z).

<img src={useBaseUrl('img/search/get-started-search/search-basics/log-query-charts-properties.png')} alt="log-query-charts-properties" style={{border:'1px solid black'}} width="800" />

> Charts will appear in new visualization style by default. If you prefer to use the classic visualizations UI, you can do so by clicking the **Classic Visualizations** button. You'll be asked to confirm your choice, Click **Switch to Classic Visualizations** to proceed.  However, if you choose a classic visualization, your selection will be remembered, and any further searches in the same or new tab will always render with the classic visualization.

Reviewer: ```suggestion
Charts will appear in **New Visualization** style by default. If you prefer to use the **Classic Visualization** UI, click the **Classic Visualizations** button, then in the popup window, click **Switch to Classic Visualizations** to proceed.  

:::tip
If you choose **Classic Visualization**, your selection will be remembered, and any further searches in the same or new tab will always render with the classic visualization.
:::
```

> ### New Visualizations

Reviewer: ```suggestion
### New Visualization
```

The **New Visualization** feature enables the new chart visualizations for your log search, allowing you customize each chart by modifying the available settings and providing a unified experience across the Sumo Logic platforms.

* **Data**. This section represents the data that has been used to plot the chart. Toggle the **Round data point** ON/OFF the round off to the value in tooltip,.

> * **Visual Settings**. These options allow you to further customize your chart: **Color Palette**, **Fill Opacity**, **Marker Type**, and **Dimensions** (x, y, and z).

Reviewer: ```suggestion
* **Visual Settings**. These options allow you to further customize your chart: [**Color Palette**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), [**Fill Opacity**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), [**Marker Type**](/docs/dashboards/panels/modify-chart/#modifydisplay-settings), and [**Dimensions**](/docs/dashboards/panels/modify-chart/#modifychart-axes) (x, y, and z).
```
